doc/metrics/lines-of-executable-code.md

```diff
@@ -0,0 +1,38 @@
+# Lines of Executable Code
+
+**Lines of Executable Code** is the approximate number of executable code lines or operations in a function of a method. Blank or comment lines are not counted. Function with high values of this metric are often complex and tough to maintain.
```

Comment:
> in a function of a method

in a function **or** a method, probably?

Reply:
@Anrock fixed